ams/server/src/main/java/com/netease/arctic/server/optimizing/OptimizingQueue.java

```diff
@@ -86,9 +86,10 @@ public OptimizingQueue(
 
   private void initTableRuntime(TableRuntimeMeta tableRuntimeMeta) {
     TableRuntime tableRuntime = tableRuntimeMeta.getTableRuntime();
+    TableOptimizingProcess process = new TableOptimizingProcess(tableRuntimeMeta);
```

Comment:
The new TableOptimizingProcess(tableRuntimeMeta) method will automatically call tableRuntime.recover(process) inside, so does this if statement still make sense?

Reply:
I have deleted it in class `TableOptimizingProcess `